templates/App/Collection/generate_label.html.twig

```diff
@@ -0,0 +1,95 @@
+{% extends 'layout.html.twig' %}
+
+{% set breadcrumb = buildBreadcrumb([{'route': 'app_collection_index', 'trans': 'menu.collections'}], collection, 'breadcrumb.add_generate_label') %}
+
+{% block title %}
+    {{ renderTitle(breadcrumb) }}
+{% endblock %}
+
+{% block breadcrumb %}
+    {{ renderBreadcrumb(breadcrumb) }}
+{% endblock %}
+
+{% block content %}
+    <div class="title-block">
+        <h1>{{ 'title.generate_label_collection'|trans({'collection': collection.title}) }}</h1>
+    </div>
+
+    <div class="content-block">
+        {{ form_start(form) }}
+        <h3 class="header col s12">Label settings</h3>
+        <div class="row">
+            <div class="col s12 l5 offset-l1 xl4 offset-xl2">
+                {{ form_row(form.labelSize, {'label' : 'label.labels_label_size', 'attr': {'data-controller': 'select--default'}} ) }}
+                {{ form_row(form.orientation, {'label' : 'label.labels_orientation', 'attr': {'data-controller': 'select--default'}} ) }}
+                {{ form_row(form.fontSize, {'label' : 'label.labels_font_size'} ) }}
+                {{ form_row(form.qrSize, {'label' : 'label.labels_qr_size'} ) }}
+                {{ form_row(form.textAlignment, {'label' : 'label.labels_text_alignment', 'attr': {'data-controller': 'select--default'}} ) }}
+            </div>
+        </div>
+
+        {% if form.selectAllItems is defined and form.object is defined %}
+        <h3 class="header col s12">Selection of items</h3>
+        <div class="row">
+            <div class="col s12 l5 offset-l1 xl4 offset-xl2">
+                <strong>{{ form_row(form.selectAllItems, {'label' : 'label.labels_select_deselect_all', 'attr': {'class': 'selectAllItems'}}) }}</strong>
+                <hr>
+            </div>
+            {% for item in form.object.children|sort %}
+                {% if loop.index is divisible by(2) %}
+                    <div class="col s12 l5 xl4">
+                {% else %}
+                    <div class="col s12 l5 offset-l1 xl4 offset-xl2">
+                {% endif %}
+                    {{ form_row(item, {'attr': {'class': 'itemCheck'}}) }}
+                </div>
+            {% endfor %}
+        </div>
+        {% endif %}
+
+        <h3 class="header col s12">Selection of fields</h3>
```

Comment:
I propose to display this only if collection has fields to avoid:
![obraz](https://github.com/museum-engineering-project/koillectionPG/assets/77413489/5da6c885-bb66-460b-8363-6bc9e7c09719)

Reply:
Good point. Done